src/api/Config/types.ts

```diff
@@ -132,6 +133,12 @@ export interface IGuestPreferencesConfig {
   uiFilter: boolean;
 }
 
+export interface WebhooksConfig {
+  deliveryFrequency?: string;
+  cleanupFrequency?: string;
+  deliveryRetentionDelayMs?: number;
```

Comment:
Isn't `cleanupFrequency` and `deliveryRetentionDelayMs` the same thing ?

Reply:
* `cleanupFrequency` is a cron expression that controls the schedule of the cleanup task. 
* `deliveryRetentionDelayMs` is the amount of milliseconds after which a delivery becomes eligible for cleanup.

So there not the same thing. For instance, we may schedule the cleanup task once per day at midnight (`0 0 * * *`) and keep completed deliveries for one week (`1000 * 60 * 60 * 24 * 7 => 604800000`).